### DISCUSSION TITLE
Standup (Tuesday, May 10, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#54] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
 ```

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Other assignments

**What are your priorities for the next 24 hours?**:
 - More MUI and React Stuff


**Shoutouts**:
 - @everyone for helping yesterday

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Not much really

**What are your priorities for the next 24 hours?**:
 - Meet with frontend team
 - Setup environment variables

**Blockers**:
 - Peña's homework

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing :O

**What are your priorities for the next 24 hours?**:
 - Check code with team

**Blockers**:
 - Peña's homework too

**Shoutouts**:
 - @CristianCazares for the dropdown menu

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Defined issues for backend development

**What are your priorities for the next 24 hours?**:
 - Complete #49 
 - Start #51 

**Blockers**:
 - None yet

**Shoutouts**:
 - @everyone for being so cool

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Defined issues with @antonieto 
 -  Corrected backend code as @CarlosCRG19  suggested

**What are your priorities for the next 24 hours?**:
 -  Start  working on #50 
 - Start #51 

**Blockers**:
 - None

**Shoutouts**:
 - @everyone for being working hard
